example/plot.py

```diff
@@ -8,7 +8,7 @@
 plt.style.use(bg_mpl_style)
 
 # load PDF data
-r, gcalc, dr, dg, gdiff = loadData("./CdSe.fgr").T
+r, gcalc, dr, dg, gdiff = loadData("example/CdSe.fgr").T

```

Comment:
please check this?  I don't see this `CdSe.fgr`?

Reply:
For some reason when I moved the files into `example` they did track. I had to do some renaming, duplicating, etc. to get them added. Should be good now.